gateway/src/apicast/policy/url_rewriting/url_rewriting.lua

```diff
@@ -28,6 +28,30 @@ local function substitute(func, subject, regex, replace, options)
   return new_uri, num_changes > 0
 end
 
+-- Returns true if the Method of the request is in the methods of the command meaning the rewrite rule should be applied
+-- Returns true if no Method is provided in the config for backwardscompatibility
+local function should_apply(methods)
+  local request_method = ngx.req.get_method()
+  local next = next
```

Comment:
I would move this next to the top of the file.

Reply:
fixed